instantiation/token.md

```diff
@@ -0,0 +1,65 @@
+# APEx instantiation API's login
+
+APEx provides secure single-sign-on for all it's instantiation services. This aids projects in implementing security
+best practices.
+
+For services with a user interface, the login should be self-explanatory. For interacting with API's directly however, an
+OIDC 'bearer token' needs to be obtained, which requires a somewhat less trivial approach. 
+
+This guide documents different approaches to generating a bearer token in a secure manner. 
+
+## Interactive login from scripts & command line
+
+The most secure login method involves an interactive login, and avoids storage of account credentials in scripts or on 
+your personal device. Therefore this is the **recommended** default method. 
+
+## Token generation via OIDC Agent
+
+The [oidc-agent](https://github.com/indigo-dc/oidc-agent) CLI tool and its corresponding Python library allow you to 
+integrate secure OIDC token generation into your scripts. The tool provides [installation instructions](https://github.com/indigo-dc/oidc-agent?tab=readme-ov-file#installation)
+for various operating systems.
+
+The following instructions show how to generate a token on a Linux operating system, as provided by the APEx user workspace.
+
+**Start oidc-agent**
+
+```bash
+eval `oidc-agent-service start`
+```
+
+**Create a configuration**
+
+Now add a configuration for your project. This requires that you request a 'client ID' from the APEx support.
```

Comment:
Is it necessary/intentional that the user has to request a client ID from support?
If it is just to execute the device flow later, APEx could provide a default client (e.g. like we do in openEO)

Reply:
I believe the client id's are per project, but not sure if that's actually needed. @netdata-be do you happen to know?

Reply:
I believe, but to be confirmed by @netdata-be, is that there is a always a client created in the a structured way. For example `<project_name>-catalogue-<dev|prod>-api`. If that is the case, there is probably no need to contact support directly.